config/application.yml.example

```diff
@@ -12,6 +12,8 @@ email_from: 'no-reply@login.gov'
 idv_max_attempts: '3'
 idv_attempt_window_in_hours: '24'
 mailer_domain_name: 'http://localhost:3000'
+max_mail_events: '4'
```

Comment:
@jgrevich @mugizico just so you know, this PR adds to ENV vars:
`mail_max_events` and `max_mail_events_window_in_days`.